photo_points/Views/Admin/PhotoStream.cshtml

```diff
@@ -18,3 +18,113 @@
 
 </form>
 
+<div class="container card-body-sm">
```

Comment:
For now I would suggest using a for loop as Brian said to display these, see my branch (just updated).